extension/.vscodeignore

```diff
@@ -1 +1,17 @@
-node_modules
+.cache/**
+.turbo/**
+.vscode-test/**
+.wdio-vscode-service/**
+coverage/**
+node_modules/**
+scripts/**
+src/**
+.eslintrc.js
+.gitignore
+.prettierignore
+.lintstagedrc.js
+.vscodeignore
+jest.config.js
+tsconfig.json
+webpack.config.ts
+yarn.lock
```

Comment:
After upgrading...

With old setup:

![image](https://user-images.githubusercontent.com/37993418/181133669-db9aa50b-c0d2-4ad6-a291-f0bab0ba5aca.png)

With this:

![image](https://user-images.githubusercontent.com/37993418/181133712-523da5e4-e587-4b96-8346-9b1b08f9ba6c.png)